src/mavlink/mavlink_camera.rs

```diff
@@ -141,8 +144,9 @@ impl Drop for MavlinkCameraComponent {
         // Remove id from used ids
         let id = self.component_id - mavlink::common::MavComponent::MAV_COMP_ID_CAMERA as u8;
         let mut vector = ID_CONTROL.lock().unwrap();
-        let position = vector.iter().position(|&vec_id| vec_id == id).unwrap();
-        vector.remove(position);
+        if let Some(position) = vector.iter().position(|&vec_id| vec_id == id) {
+            vector.remove(position);
+        }
```

Comment:
We should log when the id was not found

Reply:
https://github.com/bluerobotics/mavlink-camera-manager/issues/111